README.md

```diff
@@ -65,3 +65,61 @@ use:
 ```bash
 git commit --no-verify
 ```
+
+# Navigation menu
+
+## Supported hardware
+
+Each subsection of supported hardware should look as follows there should no
+more section
+
+```yaml
+    - 'Vendor Model':
+        - 'Overview': variants/<vendor_model>/overview.md
+        - 'Releases': variants/<vendor_model>/releases.md
+        - 'Building manual': variants/<vendor_model>/building-manual.md
+        - 'Initial deployment': variants/<vendor_model>/initial-deployment.md
+        - 'Firmware update': variants/<vendor_model>/firmware-update.md
+        - 'Recovery': variants/<vendor_model>/recovery.md
+        - 'Hardware Configuration Matrix': variants/<vendor_model>/hardware-matrix.md
+        - 'Test Matrix': variants/<vendor_model>/hardware-matrix.md
+        - 'Community Test Results (optional)': variants/<vendor_model>/community-test-results.md
+        - 'Security and Privacy (optional)': variants/<vendor_model>/security-and-privacy.md
+        - 'Other manuals (optional)': variants/<vendor_model>/other-manuals.md
+        - 'FAQ': variants/<vendor_model>/hardware-matrix.md
```

Comment:
Openness Score is missing. Question is should we mandate it or make it optional??

Reply:
Right now it is not mandatory, if you check Security and Privacy description you will see I treat Dasharo Openness Score as subsection of S&P.